scheduler.asd

```diff
@@ -13,3 +13,10 @@
   :components ((:file "scheduler")
                (:static-file "README.md")
                (:static-file "LICENSE")))
+
+(defsystem "scheduler/sqlite"
+  :depends-on ("scheduler"
+               "marshal"
+               "sqlite"
+               "iterate")
```

Comment:
iterate is used only in one place where the loop could be easily open-coded, please remove this dependency and open-code the iteration.

Reply:
Done